http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -896,6 +941,14 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       _ <- logger.info(ctx.loggingCtx)(
         s"Installing helm chart ${config.galaxyAppConfig.chart} for app ${appName.value} in cluster ${dbCluster.getGkeClusterId.toString}"
       )
+      postgresDiskNameOpt <- for {
+        disk <- getGalaxyPostgresDisk(nfsDisk.name, namespaceName, nfsDisk.googleProject, nfsDisk.zone)
```

Comment:
Do we need to call `getGalaxyPostgresDisk` (which makes a Google call) here? Since we just need the name could we do:
```
postgresDiskName = getGalaxyPostgresDiskName(nfsDisk.name)
```

Reply:
this is to support disks that're created in old naming convention...altho I'm not sure how important that is given restore from a disk is not available in UI yet

Reply:
Right, I see… yeah technially disk support is not in the UI yet but probably safe to have this here I suppose.

Reply:
yeah I wasn't sure if my change or Gabriela's was going to get out first so figured better safe than sorry lol and will protect against any folks who maybe made a PD and stopped using it awhile will be able to start up again no problem once those UI changes do go out.